config/environments/development.rb

```diff
@@ -152,7 +148,7 @@
   # http://www.example.com/markus/svn/Repository_Name. Make sure the path
   # after the hostname matches your <Location> directive in your Apache
   # httpd configuration
-  REPOSITORY_EXTERNAL_BASE_URL = 'http://www.example.com/markus/svn'
+  REPOSITORY_EXTERNAL_BASE_URL = "http://www.example.com/markus/svn"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.